_scripts/templates/Go/Test.go

```diff
@@ -7,7 +7,7 @@ import (
     "io"
     "time"
     "github.com/antlr/antlr4/runtime/Go/antlr"
-    "example.com/myparser/parser"
+    "example.com/myparser/<package_name>"
```

Comment:
Whiel the default "package" generated by the Antlr4 tool is "parser", trgen now accepts a package spec in the pom. For now, CI just let's it be "parser", so the generated code is the same. However, there is a big problem in the Antlr4 tool for Go: one cannot have multiple parsers or lexers in one directory. trgen does not determine how to partition the parser grammars and associated source code into separate directories, and I would prefer that Antlr just handle this by qualifying the names of global variables in the code that it generates with the name of the grammar.